src/main/java/codeu/controller/RegisterServletTest.java

```diff
@@ -16,22 +17,30 @@
     private HttpServletRequest mockRequest;
```

Comment:
Consider making this class use the MockitoJUnitRunner (see here: https://stackoverflow.com/questions/15494926/initialising-mock-objects-mockito).  That way, you won't have to explicitly mock each instance, just mark as @Mock.  Note that your imports will have to include the runner and the Mock annotation, usually done with something like "import static org.mockito.Mock;"